BUILD

```diff
@@ -12,9 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@io_bazel_rules_go//go:def.bzl", "gazelle", "go_prefix")
-
-go_prefix("github.com/bazelbuild/bazel-watcher")
```

Comment:
In order to remove this you have to add a `# gazelle:prefix github.com/example/project` in its place

Reply:
Actually, you don't need that because gazelle is already called with the prefix [here](https://github.com/bazelbuild/bazel-watcher/pull/137/files#diff-96239ee297e0a92ac6ff96a6bc434ef0L21).